app/Exceptions/Handler.php

```diff
@@ -34,34 +35,36 @@ public function report(Exception $e)
     /**
      * Render an exception into an HTTP response.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Exception  $e
+     * @param  \Illuminate\Http\Request $request
+     * @param  \Exception $e
      * @return \Illuminate\Http\Response
      */
     public function render($request, Exception $e)
     {
-        if (env('APP_DEBUG') != true) {
```

Comment:
Ah, I see. Removing the APP_DEBUG clause in the Handler actually mutes all Laravel errors, not just errors occurring in the path. It's much easier to fix bugs and work on new features when you can see Laravel's debug page.